master/usage/configuration/node.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Configuring a node IP address and subnet
+---
+
+Each Calico node has key routing configuration that may be specified in a number of
+different ways.  In particular, this includes the IPv4 and IPv6 addresses and 
+subnets used for routing.
+
+To configure the IP address and subnet of your node, you have the options of:
```

Comment:
I think you should explicitly say that these values are detected and stored during node startup